semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/nodemethods/MethodReturnMethods.scala

```diff
@@ -0,0 +1,16 @@
+package io.shiftleft.semanticcpg.language.nodemethods
+
+import io.shiftleft.codepropertygraph.generated.nodes
+import io.shiftleft.semanticcpg.language._
+import io.shiftleft.semanticcpg.language.types.expressions.ControlStructure
+import io.shiftleft.semanticcpg.language.types.expressions.generalizations.AstNode
+import io.shiftleft.semanticcpg.language.types.structure.{Local, MethodParameter}
+import scala.jdk.CollectionConverters._
+import io.shiftleft.Implicits.JavaIteratorDeco
+
+class MethodReturnMethods(val node: nodes.MethodReturn) extends AnyVal {
+
+  def formalReturnToReturn: Seq[nodes.Return] =
```

Comment:
Just "toReturn" is enough since the in the DSL we know that the expansion source is a method return.
Please also add a version of this expansion to the DSL.

Reply:
done in 3353280